episodes/narrative.md

```diff
@@ -114,6 +114,12 @@ GNU offers commentary about a variety of licenses for free software; this resour
 
 ## Finding a Dataset
 
+:::::::::::::::::: testimonial
+
+> It was very distracting to me in a workshop, when we were given a dataset to work with but no-one explained what that data was and what it meant. It prevented me from following what the instructors were actually trying to teach me, while I tried to figure out the meaning of the numbers, columns names etc on my own.
```

Comment:
This is a nice addition!  Do you want to give attribution?

Reply:
I'm seeing the original issue doesn't have attribution as well.  We could do "- Previous CLDT Learner" or if toby remembers who this is ask if they would like to be attributed.

Reply:
I believe we got permission to use the quote anonymously.